README.NewResourceFromTerraform.md

```diff
@@ -618,8 +639,8 @@ update to date, we need to add the new CRDs and samples to existing doc.
 1.  Update
     [scripts/generate-google3-docs/resource-reference/overview.md](scripts/generate-google3-docs/resource-reference/overview.md)
     by adding a row for your resource to the resource table. Note you need to
-    use a proper template variable for your new service. TODO: Provide guidance
-    on how to find the variable name.
+    reach out to the Config Connector team explicitly via the PR comments to
```

Comment:
Instead of mentioning the "team", which isn't that obvious to contributors, it might be easier to point them to the owners:

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/OWNERS

Reply:
Good point. Done!